src/middlewares/auth/utils/getResourceBusinessId/getResourceBusinessIdByCategoryId.js

```diff
@@ -0,0 +1,14 @@
+const { BusinessClass } = require("../../../../models");
+
+const getResourceBusinessIdByCategoryId = async (req, res) => {
+  let { categoryId } = req.params;
+  //console.log(categoryId);
+
+  let category = await BusinessClass.findById(categoryId);
```

Comment:
How can you find category in BusinessClass Model?

Reply:
it was a typo

Reply:
now resolved